src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/SourceGenerator.cs

```diff
@@ -19,7 +22,7 @@
 namespace CSharpSyntaxGenerator
 {
     [Generator]
-    public sealed class SourceGenerator : ISourceGenerator
+    public sealed class SourceGenerator : CachingSourceGenerator
```

Comment:
Why are we only doing this for C# and not also VB?

Reply:
We don't have a VB generator yet.